ex03.exs

```diff
@@ -60,9 +60,35 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    ceil_div = fn dividend, divisor -> 
+                Integer.floor_div(dividend, divisor) + 1
+               end
+
+    chunk_size = Enum.count(collection) |> ceil_div.(process_count)
+    assign_tasks = fn chunk -> 
+                     Task.async(Enum, :map, [ chunk, function ])
+                   end
+    
+    gather_results = fn task -> 
+                       Task.await(task) 
+                     end
+    
```

Comment:
Using functions assigned to variable with meaningful names makes this pipeline really easy to read. Nice